01-basics/03-MarkedList/script.js

```diff
@@ -29,4 +29,18 @@ const emails = [
   'Isaias_Kuhic@jarrett.net',
 ];
 
-// Требуется создать Vue приложение
+const app = createApp({
+  data() {
+    return {
+      emails,
+      search: null,
+    }
+  },
+  computed: {
+    filteredEmails() {
+      return this.emails.filter(el => el.includes(this.search));
+    }
+  }
```

Comment:
On the webinar, we used computed to prepare **filtered** data and displayed **filtered meetups list**. But this task is different. Here you need to display **all** the emails, but make some items **marked**.

To archive that, you need to create a computed property with a marked list. For example, you can compute a list of objects where both the email string and `isMarked` boolean flag are stored.

Reply:
Agree,  created a list of objects (as a computed property) where both the email string and ```isMarked``` boolean flag are stored.